server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -0,0 +1,522 @@
+              onChange={this.handleFieldChange('transition')}
+              type="radio"
+              value={transition.transition}
+            />
+            <label
+              htmlFor={`transition-${transition.transition}`}
+              style={{ float: 'none', display: 'inline', left: 0, cursor: 'pointer' }}>
+              {translate('issue.transition', transition.transition)}
+            </label>
+            {this.renderAffected(transition.count)}
+            <br />
+          </span>
+        ))}
+      </div>
+    );
+  };
+
+  renderCommentField = () => {
+    const affected: number = this.state.issues.filter(hasAction('comment')).length;
+
+    if (affected === 0) {
+      return null;
+    }
+
+    return (
+      <div className="modal-field">
+        <label htmlFor="comment">
+          {translate('issue.comment.formlink')}
+          <Tooltip overlay={translate('issue_bulk_change.comment.help')}>
+            <i className="icon-help little-spacer-left" />
+          </Tooltip>
+        </label>
+        <div>
+          <textarea
+            id="comment"
+            onChange={this.handleFieldChange('comment')}
+            rows="4"
+            style={{ width: '100%' }}
+            value={this.state.comment || ''}
+          />
+        </div>
+        <div className="pull-right">
+          <MarkdownTips />
+        </div>
+      </div>
+    );
+  };
+
+  renderNotificationsField = () => (
+    <div className="modal-field">
+      <label htmlFor="send-notifications">{translate('issue.send_notifications')}</label>
+      {this.renderCheckbox('notifications')}
+    </div>
+  );
+
+  renderForm = () => {
+    const { issues, paging, submitting } = this.state;
+
+    const limitReached: boolean = paging != null &&
+      paging.total > paging.pageIndex * paging.pageSize;
+
+    return (
+      <form id="bulk-change-form" onSubmit={this.handleSubmit}>
+        <div className="modal-head">
+          <h2>{translateWithParameters('issue_bulk_change.form.title', issues.length)}</h2>
+        </div>
+
+        <div className="modal-body">
+          {limitReached &&
+            <div className="alert alert-warning">
+              {translateWithParameters('issue_bulk_change.max_issues_reached', issues.length)}
+            </div>}
+
+          {this.renderAssigneeField()}
+          {this.renderTypeField()}
+          {this.renderSeverityField()}
+          {this.renderAddTagsField()}
+          {this.renderRemoveTagsField()}
+          {this.renderTransitionsField()}
+          {this.renderCommentField()}
+          {this.renderNotificationsField()}
+        </div>
+
+        <div className="modal-foot">
+          {submitting && <i className="spinner spacer-right" />}
+          <button disabled={submitting} id="bulk-change-submit">{translate('apply')}</button>
+          {this.renderCancelButton()}
+        </div>
+      </form>
+    );
+  };
+
+  render() {
+    return (
+      <Modal
```

Comment:
When using arrow keys in the different select it's conflicting with the shortcuts of the issues. So keyboard events should probably not be propagated.